templates/xls.html

```diff
@@ -17,7 +17,25 @@ <h5>Delete measurements from an XLS file</h5>
         </form>
     </div>
 
-    {% for log_entry in log %}
-        {{ log_entry }}<br>
-    {% endfor %}
+    {% if log %}
+    <div class="row mt-4">
+        <div class="col-md-12">
+        <h3>Upload results {% if error_count != 0 %}&mdash; <span class="badge badge-warning">there were errors during the import</span>{% endif %}</h3>
+        <ul class="list-group">
```

Comment:
maybe some small margin-bottom will be good
<img width="631" alt="image" src="https://user-images.githubusercontent.com/10351924/82471286-6825e880-9abe-11ea-8a76-cb40ddf49776.png">